README.md

```diff
@@ -35,7 +35,8 @@ You will need an authorization token to create a channel on Kolibri Studio. In o
 
 ### Step 2: Initializing the Channel ###
 
-To run the Ricecooker, you must include a `construct_channel` method in your sushi chef file that returns a ChannelNode object. This function will be responsible for building the structure of your channel.
+To run the Ricecooker, you must include `create_channel` and `construct_channel` methods in your sushi chef file.
```

Comment:
Is the main goal of this to get the channel id as soon as possible? It might be possible to just get the source_domain and source_id to calculate the channel_id (maybe have the user pass them in as parameters?)